canvas_modules/harness/cypress/support/canvas/keyboard-cmds.js

```diff
@@ -74,3 +74,10 @@ Cypress.Commands.add("useShiftKey", () => "{shift}");
 
 // Press 'backspace' key on keyboard
 Cypress.Commands.add("useBackspaceKey", () => "{backspace}");
+
+
+// Press cmd+v key on keyboard to paste the nod
+Cypress.Commands.add("moveMouseToCoordinates", (x, y) => {
+	cy.get(".d3-svg-canvas-div").trigger("mousemove", x, y);
+});
+
```

Comment:
This function shouldn't be in the keyboard-cmds.js file. It is nothing to do with the keyboard operation. Having said that it doesn't really belong in node-cmds.js either, which is where the original function was located.  Please create a new file called operation-cmds.js and put it there. Don't forget to add the copyright notice the same as the other files.